model_prices_and_context_window.json

```diff
@@ -286,6 +286,33 @@
         "mode": "chat"
     },
     "ft:gpt-3.5-turbo": {
+        "max_tokens": 16385,
```

Comment:
`max_tokens` maps to `max_output_tokens`. It's the openai parameter

Reply:
See - https://github.com/BerriAI/litellm/blob/2797b30a504916466dfcb0be68bca605a72c6392/model_prices_and_context_window.json#L3

Reply:
I didn't change the `ft:gpt-3.5-turbo` entry, but it definitely looks odd. In any case, there is no actualy `ft:gpt-3.5-turbo:<suffix>` model to use in OpenAI. Unless it gets automatically mapped to `ft:gpt-3.5-turbo-0125:<suffix>`, we could drop it from pricing